perf/renaissance-jmh/gen_playlist.py

```diff
@@ -45,7 +45,7 @@
 
     file.write("\t\t<testCaseName>renaissance-jmh-" + bench + "</testCaseName>\n")
     file.write("\t\t<command>\n")
-    file.write("\t\t\t$(JAVA_COMMAND) $(ADD_OPENS_CMD) $(JVM_OPTIONS) -Xlog:gc*:file=" + bench + "_gc.log -XX:+UseG1GC -Xms12G -Xmx12G -XX:ThreadPriorityPolicy=1 -XX:+AlwaysPreTouch -XX:+UseLargePages -XX:+UseTransparentHugePages")
+    file.write("\t\t\t$(JAVA_COMMAND) $(ADD_OPENS_CMD) $(JVM_OPTIONS) -Xlog:gc*,gc+ref=debug,gc+phases=debug,gc+age=trace,gc+ergo*=trace,safepoint:file=gc.log" + bench + "_gc.log -XX:+UseG1GC -Xms12G -Xmx12G -XX:ThreadPriorityPolicy=1 -XX:+AlwaysPreTouch -XX:+UseLargePages -XX:+UseTransparentHugePages")
```

Comment:
remove gc.log as bench+"_gc.log" is already takes care of passing gc file specific to bench.
gc+ref=debug,gc+phases=debug,gc+age=trace,gc+ergo*=trace,safepoint:file=**gc.log**